cfme/login.py

```diff
@@ -72,6 +71,12 @@ def log_in(self, user, method='click_on_login'):
             'password': user.credential.secret,
         })
         self.submit_login(method)
+        logged_in_view = self.browser.create_view(BaseLoggedInPage)
+        if logged_in_view.logged_in:
+            name = logged_in_view.current_fullname
+            if name != user.name:
+                self.logger.info('setting the appliance.user.name to %r', name)
```

Comment:
Unless we require that the User object passed via login must have name, what else can we do ... I can put hte none comparison instead